app/assets/javascripts/admin/controllers/admin-users-list-show.js.es6

```diff
@@ -43,7 +48,19 @@ export default Ember.Controller.extend({
     var self = this;
     this.set('refreshing', true);
 
-    AdminUser.findAll(this.get('query'), { filter: this.get('listFilter'), show_emails: this.get('showEmails') }).then(function (result) {
+    AdminUser.findAll({ filter: this.get('listFilter'), show_emails: this.get('showEmails') }).then(function (result) {
```

Comment:
We prefer using [arrow functions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions) now :)